fetch/api/response/json.any.js

```diff
@@ -0,0 +1,14 @@
+// See also /xhr/json.any.js
+
+promise_test(async t => {
+  const response = await fetch(`data:,\uFEFF{ "b": 1, "a": 2, "b": 3 }`);
+  const json = await response.json();
+  assert_array_equals(Object.keys(json), ["b", "a"]);
+  assert_equals(json.a, 2);
+  assert_equals(json.b, 3);
+}, "Ensure the correct JSON parser is used");
+
+promise_test(async t => {
+  const response = await fetch("/xhr/resources/utf16-bom.json");
+  return promise_rejects_js(t, TypeError, response.json());
```

Comment:
SyntaxError?

Reply:
Thanks! I must have mixed up URLs locally as I thought this was passing in Firefox and failing in Chrome, but I guess it was failing in both as now they both pass...